stdlib/private/TensorFlowUnittest/TensorFlowUnittestRemote.swift

```diff
@@ -0,0 +1,109 @@
+#if os(macOS) || os(iOS) || os(watchOS) || os(tvOS)
+import Darwin
+#else
+import Glibc
+#endif
+import CTensorFlow
+import CTensorFlowUnittest
+import TensorFlow
+import StdlibUnittest
+
+typealias CTFServer = OpaquePointer
+
+@usableFromInline
+func debugLog(_ message: @autoclosure () -> String,
+              file: StaticString = #file,
+              line: UInt = #line) {
+  if _RuntimeConfig.printsDebugLog {
+    print("[\(file):\(line)] \(message())")
+    // This helps dump more log before a crash.
+    fflush(stdout)
+  }
+}
+
+
+private class TestCluster {
+  var num_tasks:Int = 0
+  var servers: [CTFServer] = []
+  public var serverDef:String = ""
+
+  public init(num_tasks: Int) {
+    self.num_tasks = num_tasks
+    self.serverDef = startServers()
+  }
+
+  deinit {
+    // TODO: There is no clean way to shutdown GrpcServers yet.
+  }
+
+  private func getServerDef(tasks: String, task_index: Int) -> String {
+    return  """
+      cluster {
+        job {
+          name: "localhost"
+          \(tasks)
+        }
+      }
+      job_name: "localhost"
+      task_index: \(task_index)
+      protocol: "grpc"
+      """
+  }
+
+  private func getTasks() -> String {
+    var tasks = ""
+    for i in 0..<num_tasks {
+      let port = TF_PickUnusedPortOrDie()
+      let task = """
+          tasks {
+            key: \(i)
+            value: "localhost:\(port)"
+          }
+      """
```

Comment:
Is this indentation intentional? Note that the indentation of a multi-line string literal is with respect to the closing `"""`. So this would be 4-column indented.

Reply:
Yes, it is intentional and mostly for aesthetic reasons as this would go into cluster def.